lib/bosh/BoshDirectorClient.js

```diff
@@ -432,7 +434,16 @@ class BoshDirectorClient extends HttpClient {
         }
       })
       .then(manifest => {
-        return manifest.properties.agent;
+        if (manifest.instance_groups) {
+          let agentJob = {};
+          _.each(manifest.instance_groups, (instance_group) => {
+            agentJob = _.find(instance_group.jobs, job => job.name === 'broker-agent');
```

Comment:
can you store 'broker-agent' in constants ?

Reply:
ok..